### DISCUSSION TITLE
keepassxc: cannot access ssh-agent socket

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#6529] [Q&A] [answered]
On firejail latest git build (I don't have this issue with the latest released version), KeepassXC is not seeing ssh agent. In keepassxc.local, the following was used to allow for ssh-agent: `include allow-ssh.inc`. Any ideas? Running KeepassXC without firejail works as expected. `SSH_AUTH_SOCK=/run/user/1000/ssh-agent.socket`.

**Comment by kmk3** [comment, marked as answer]
> On firejail latest git build (I don't have this issue with the latest
> released version), KeepassXC is not seeing ssh agent.

keepassxc uses whitelisting in `${RUNUSER}` since the following PR:

* #6391

To allow the default ssh-agent socket path, it should be enough to add
following to keepassxc.local:

```
noblacklist /tmp/ssh-*
whitelist /tmp/ssh-*
```

> `SSH_AUTH_SOCK=/run/user/1000/ssh-agent.socket`.

If you use a custom socket path, you need to allow that instead:

```
noblacklist ${RUNUSER}/ssh-agent.socket
whitelist ${RUNUSER}/ssh-agent.socket
```

**Comment by kmk3** [comment]
@rusty-snake

Thoughts on allowing access to the ssh-agent socket by default on
keepassxc.profile or adding a profile comment?

**Reply by rusty-snake** [reply]
Allowing ssh-agent by default should be fine IMO.

**Reply by kmk3** [reply]
Done:

* #6531
